jmin-jlink/src/MANIFEST.MF

```diff
@@ -0,0 +1 @@
+Main-Class: hellomodule.HelloWorld
```

Comment:
I don't see this used anywhere: what's its purpose?

Reply:
This is referenced in the README

> #### Create a hellomodule jar (i.e. to use with jarmin)
> ```
> <PATH_TO_JDK>/bin/jar \
> 	--create \
> 	--file=build/hellomodule.jar \
> 	--manifest=src/MANIFEST.MF \             <----------- here
> 	-C build/modules/hellomodule module-info.class \
> 	-C build/modules/hellomodule hellomodule
> ```